app/controllers/helpers_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class HelpersController < ApplicationController
+  before_action :set_initiative, only: %i[index create destroy]
+
+  def index
+    render json: @initiative.helpers
+  end
+
+  def create
+    new_helpers = @initiative.helpers.concat helpers_from_body
+    if @initiative.update(helpers: new_helpers)
+      render json: @initiative.helpers, status: :created, location: @initiative
+    else
+      render json: @initiative.errors, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    new_helpers = @initiative.helpers - [Wyeworker.find(params[:id])]
+    if @initiative.update(helpers: new_helpers)
```

Comment:
Same as above, `@initiative.helpers.delete(Wyeworker.find(params[:id]))` should suffice (with the save in the if)